### ISSUE TITLE
Group 1 is not created when buffer size < 256.

### ISSUE THREAD

**Issue by marierm** [supercollider/supercollider#3492]
It looks like Group 1 is not created whenever I try to use a buffer size < 256.  This is on Arch Linux, using SC 3.9.

First, start Jack with a buffer size of 64 samples, then fire up SC.
```

s.boot;
s.queryAllNodes;
// NODE TREE Group 0

g = Group.head(s);
// -> Group(1001)
// FAILURE IN SERVER /g_new Group 1 not found
```

When Jack is running with a buffer size of 256, I do not have this
issue.

```
s.boot;
s.queryAllNodes;
// NODE TREE Group 0
//    1 group

g = Group.head(s);
// -> Group(1001)
```

**Comment by jamshark70** [comment]
May I recommend not multiple-posting? Same issue on sc-users, sc-dev and here. I don't know where I should reply.

I expect this will be sensitive not only to settings but to specific hardware.

**Comment by patrickdupuis** [comment]
I agree. In this case, we should reply on mailing list. We can use this issue when to discuss the actual fix if/when the issue gets identified.

**Comment by patrickdupuis** [comment]
I've been able reproduce this issue, although not with 100% consistency, by following what @marierm posted here: http://new-supercollider-mailing-lists-forums-use-these.2681727.n2.nabble.com/Group-1-is-not-created-when-buffer-size-256-tp7637770p7637845.html
